creator-node/src/services/sync/primarySyncFromSecondary.js

```diff
@@ -306,6 +307,8 @@ async function saveEntriesToDB({ fetchedCNodeUser, logger, logPrefix }) {
 
       cnodeUserUUID = localCNodeUser.cnodeUserUUID
 
+      fixUserIfInconsistent({ localCNodeUser, fetchedCNodeUser, logger })
```

Comment:
why put this here? maybe we should put this in the catch in case of error

Reply:
It seemed to make sense here since it seemed right to validate the local cnodeuser the moment this function realizes one exists. The `catch` in this function was super generic so it didn't necessarily mean that a user was inconsistent. 

I can move it to the catch though if it makes more sense in the broader scope of things.